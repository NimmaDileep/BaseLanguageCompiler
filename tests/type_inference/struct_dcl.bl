struct Dog {
    age: Int,
    name: String,
    isHappy: Boolean
}

print(Dog(5, "Fido", true));

var dog = Dog(3, "Spot", false);

print(dog.age);
print(dog.name);
print(dog.isHappy);