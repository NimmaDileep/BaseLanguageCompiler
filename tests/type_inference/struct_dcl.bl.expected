Dog{age=5,name=Fido,isHappy=true}
3
Spot
false